### ISSUE TITLE
Improve README?

### ISSUE THREAD

**Issue by ejgallego** [LLM4Coq/pytanque#4]
Hi folks,

it seems to me the description and README of this repos are hard to parse by people not familiar with what `petanque` is.

IMHO, we could omit `petanque` at all from the description, as most users of `petanque` will use `pytanque`, and instead describe the project with some phrase such as: 

"Pytanque is a Python API for lightweight communication with the Rocq proof assistant, with applications on machine learning and software engineering"

instead of

"Lighweight communication with Petanque"

WDYT?

**Comment by gbdrt** [comment]
Yes that is a good idea.
